### PR TITLE
Added the Interaction Bar

### PR DESCRIPTION
**Pull request by Riley142**
Just need to add animation for the heart, but now I'll add it to "Nice to have"

### CODE REVIEW

**Review by jadavey91 — APPROVED** [review]
Looks great!

**Review by lisarawlings — APPROVED** [review]
looks good

File: app/src/Components/InteractionBar.js [hunk 1]
```diff
@@ -1,31 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
 import "./InteractionBar.css";
 
 const InteractionBar = () => {
+  const [commentCount, setCommentCount] = useState(1);
+  const [retweetCount, setRetweetCount] = useState(5);
+  const [likeCount, setLikeCount] = useState(100);
+  const [shareCount, setShareCount] = useState(10);
+
```
Comment (boothscript) [inline app/src/Components/InteractionBar.js:9]: The Tweet component will have the count data init, maybe the interactionBar could except those as props?

File: app/src/Components/InteractionBar.js [hunk 2]
```diff
@@ -1,31 +1,42 @@
...
-        <p>5</p>
+        <button type="button" onClick={() => setShareCount(shareCount + 1)}>
+          <i className="ri-download-2-line"> </i>
+        </button>
+        <p> {shareCount} </p>
       </div>
-      
     </div>
   );
 };
```
Comment (boothscript) [inline app/src/Components/InteractionBar.js:42]: Maybe this could be refactored to one icon component that gets reused in the future?

**Review by boothscript — APPROVED** [review]
A couple of comments/ideas for the future but its looks great.

File: app/src/Components/InteractionBar.js (hunk 2 above)
Comment (boothscript) [reply app/src/Components/InteractionBar.js:42]: Sorry, this is supposed to be a comment on all the lines above (interactionBar return JSX)

**Pull request merged**